sources/packages/backend/libs/integrations/src/services/institution-location/institution-location.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from "@nestjs/common";
+import { InstitutionLocation, RecordDataModelService } from "@sims/sims-db";
+import { DataSource } from "typeorm";
+
+/**
+ * Institution location service.
+ */
+@Injectable()
+export class InstitutionLocationService extends RecordDataModelService<InstitutionLocation> {
+  constructor(dataSource: DataSource) {
+    super(dataSource.getRepository(InstitutionLocation));
+  }
+
+  /**
+   * Get all the institution codes of the institutions who
+   * are enabled for integration.
+   * @returns institution codes.
+   */
+  async getAllIntegrationEnabledInstitutionCodes(): Promise<string[]> {
+    const locations = await this.repo.find({
+      select: {
+        id: true,
```

Comment:
if the return is maping to get only the `institutionCode`. then do we need line 22

Reply:
Id removed.